### DISCUSSION TITLE
Request

### DISCUSSION THREAD

**Discussion by arinda20** [basoro/Khanza-Lite#149] [Ideas]
Mohon ditambahkan modul laporan, yang bisa untuk export data pasien dan pemeriksaan ralan dalam bentuk excel

**Comment by basoro** [comment]
Silahkan lihat di 

https://github.com/basoro/mlite/commit/845b9eabf6cd2a72c249bbb32200389e58b1a4b7

Atau update manual di

https://github.com/basoro/mlite/tree/845b9eabf6cd2a72c249bbb32200389e58b1a4b7

Salam
